src/Expression/Logical/Not.php

```diff
@@ -9,6 +9,9 @@
 use Arkitect\Expression\NegativeDescription;
 use Arkitect\Rules\Violations;
 
+/**
+ * @deprecated deprecated
+ */
 class Not implements Expression
```

Comment:
Why deprecate and not delete? We found that is misleading, so for me it's better remove it.

Reply:
Yes, this is a BC and I tried a different approach before delete it completely. 
I'll remove it completely.

Reply:
As of now Not it is just not working as expected, we should get rid of it

Reply:
I have Removed the class `Not`